CMakeLists.txt

```diff
@@ -19,6 +29,10 @@ if (VITA_BUILD)
     set(VITA_VERSION "00.10")
 endif()
 
+if (SWITCH_BUILD)
+    set(SWITCH_VERSION "1.02")
```

Comment:
By the way, I see that the VITA version is hardcoded to some (other) value. I think that we should stick with the PC version in the future.

Reply:
fixed in 
https://github.com/bvschaik/julius/pull/124/commits/4e3c8b084b64d8ccde179fe6d8cfb3027195920c